modules/local/create_blobdir.nf

```diff
@@ -0,0 +1,49 @@
+process CREATE_BLOBDIR {
+    tag "$meta.id"
+    label 'process_single'
+
+    if (params.enable_conda) {
+        exit 1, "Conda environments cannot be used when using the CREATE_BLOBDIR module. Please use docker or singularity containers."
+    }
+    container "genomehubs/blobtoolkit-blobtools:3.3.4"
+
+    input:
+    tuple val(meta), path(window, stageAs: 'dir/*')
+    tuple val(meta), path(busco)
+    tuple val(meta), path(blastp)
+    tuple val(meta), path(yaml)
+    path(taxdump)
+    val(genome_accession)
+
+    output:
+    tuple val(meta), path('**/*meta.json')              , emit: json
+    tuple val(meta), path('**/*identifiers.json')       , emit: identifiers
+    tuple val(meta), path('**/*buscogenes_phylum.json') , emit: buscogenes_phylum
+    tuple val(meta), path('**/*cov.json')               , emit: coverage
+    tuple val(meta), path('**/*busco.json')             , emit: busco
```

Comment:
That's very interesting. Here you break down all the content of the "blob dir" into different channels instead of considering the whole directory as a single output.
In his "view sub-workflow" (#47), @alxndrdiaz treats the blob dir as a single directory that is modified in place. I had reservations about that because it doesn't fit Nextflow's paradigm very well.

I much prefer your way of doing, @zb32 , but that will require the following changes to the consumers / processors of the blob dir.
1. We'll need to know precisely what each of them consumes. For instance, if something only uses the busco data, its module may only take `buscogenes_phylum` and `busco` as inputs, etc.
2. All those inputs would have to be put together into the same directory, likely with a `stageAs` directive – essentially recreating a minimal blob dir on the fly
3. If the blobdir is required as a single output of the pipeline, we'll have to configure publishDir appropriately for all the modules that contribute to it.

Reply:
I can co-ordinate with @alxndrdiaz for this since he has the PR for the modules that uses `blobdir` :D

Reply:
> I can co-ordinate with @alxndrdiaz for this since he has the PR for the modules that uses `blobdir` :D

The module outputs the expected files as described in [BlobDir specification](https://github.com/blobtoolkit/specification), stageAs directive or a local module could be used to put these files together in a single directory as long as it has a valid `DatasetID` as name (I am not completely sure about this: but it allows a prefix as in the `config.yaml `file or` accession`). This should be done before passing `BlobDir` as input for the `view `subworkflow.